identity-service/src/awsSNS.js

```diff
@@ -44,7 +44,8 @@ function _formatIOSMessage(
   targetARN,
   badgeCount,
   playSound = true,
-  title = null
+  title = null,
+  notification
```

Comment:
Let's put `notification` above the optional arguments!

Also I noticed the jsdoc comment is wayyy off 😅

Reply:
haha good point, ill fix that